redis/src/commands/mod.rs

```diff
@@ -2432,7 +2432,7 @@ implement_commands! {
     /// XREVRANGE key + -
     /// ```
     /// [Redis Docs](https://redis.io/commands/XREVRANGE").arg(key).arg("+").arg("-)
-    fn xrevrange_all<K: ToRedisArgs>(key: K) -> (Vec<Value>) {
+    fn xrevrange_all<K: ToRedisArgs>(key: K) -> (streams::StreamRangeReply) {
```

Comment:
`xrevrange_all` wasn't behind the `streams` feature gate

```suggestion
    #[cfg(feature = "streams")]
    #[cfg_attr(docsrs, doc(cfg(feature = "streams")))]
    fn xrevrange_all<K: ToRedisArgs>(key: K) -> (streams::StreamRangeReply) {
```

Reply:
thanks!